src/resources/dicts/en.edn

```diff
@@ -221,6 +221,23 @@
  :page/updated-at "Updated At"
  :page/backlinks "Backlinks"
  :linked-references/filter-search "Search in linked pages"
+ :linked-references/unexpected-error "Linked References: Unexpected error. Please re-index your graph first."
+ :linked-references/filter-heading "Filter"
+ :linked-references/filter-directions "Click to include and shift-click to exclude. Click again to remove."
+ :linked-references/filter-includes "Includes: "
+ :linked-references/filter-excludes "Excludes: "
+ :linked-references/reference-count (fn [filters? filter-n total]
```

Comment:
Let's simplify this fn as it only needs 2 args, the filtered-count and total-count, as we can pass nil for filtered-count if filters? isn't present in the calling fn. Since most translators aren't coders would be good to show examples in comments of different strings one can see